helmnet/hybridnet.py

```diff
@@ -276,13 +320,31 @@ def test_step(self, batch, batch_idx):
                 return_wavefields=True,
                 return_states=False,
             )
-            # Get loss
-            losses = [self.test_loss_function(x) for x in output["residuals"]]
-            losses = torch.stack(losses, 1)
-            return {
-                "losses": losses,
-                "wavefields": [x.cpu() for x in output["wavefields"]],
-            }
+        # Get loss
```

Comment:
Should this be intented, to stay in the scope of `torch.no_grad()`, or perhaps lightning already takes care of this?

Reply:
Seems so, will remove in next commit. https://github.com/PyTorchLightning/pytorch-lightning/issues/2171

Reply:
Oh ok, if it does then is probably right as it is!